include/libtorrent/bandwidth_limit.hpp

```diff
@@ -33,30 +33,30 @@ POSSIBILITY OF SUCH DAMAGE.
 #ifndef TORRENT_BANDWIDTH_CHANNEL_HPP_INCLUDED
 #define TORRENT_BANDWIDTH_CHANNEL_HPP_INCLUDED
 
-#include "libtorrent/aux_/disable_warnings_push.hpp"
+#include "libtorrent/assert.hpp"
 
-#include <boost/integer_traits.hpp>
+#include "libtorrent/aux_/disable_warnings_push.hpp"
 #include <boost/cstdint.hpp>
-
 #include "libtorrent/aux_/disable_warnings_pop.hpp"
 
-#include "libtorrent/assert.hpp"
+#include <limits>
 
 namespace libtorrent {
 
 // member of peer_connection
 struct TORRENT_EXTRA_EXPORT bandwidth_channel
 {
-	static const int inf = boost::integer_traits<int>::const_max;
+    static boost::int32_t const inf;
```

Comment:
ah, right. numeric_limits::max() is not constexpr in C++98. the previous version here seems simpler, doesn't it? to use ``boost::integer_traits``